src/tests/system/tests/test_passkey.py

```diff
@@ -82,7 +82,7 @@ def test_passkey__register__ipa(ipa: IPA, moduledatadir: str, testdatadir: str):
 @pytest.mark.importance("critical")
 @pytest.mark.topology(KnownTopologyGroup.AnyProvider)
 @pytest.mark.builtwith(client="passkey", provider="passkey")
-def test_passkey__su(client: Client, provider: GenericProvider, moduledatadir: str, testdatadir: str):
+def test_passkey__su_user(client: Client, provider: GenericProvider, moduledatadir: str, testdatadir: str):
```

Comment:
The other PR had name changes for tests before this one in test_passkey.py as well.   Can those be added here?